stack.yaml

```diff
@@ -0,0 +1,61 @@
+resolver: lts-5.13
```

Comment:
All right, I'll take a crack at writing a script that generates the stack.yaml file. This will be useful for both 1) lazy people like me who don't always keep our repository clones in-sync, and 2) so that the server can test on the latest versions of the repos every day.